controls/V-73063.rb

```diff
@@ -75,7 +75,7 @@
 )
 
 PG_VERSION = attribute(
-  'pg_version',
+  'pg_versionsion',
```

Comment:
Typo ?

Reply:
Copy-paste error; this has been corrected.